### ISSUE TITLE
UV_Coords mode: viewprojection

### ISSUE THREAD

**Issue by ggml** [everoneishappy/InstanceNoodles#6]
Hello,
Was looking to build an 'adaptive subdivision' patch that would tesselate by depth,
but it needs camera projected UV coordinates.
Could this make for an added mode to UV_Coords, with viewprojection input ?
It could make control textures camera-aware, also help at projeciton-mapping

thank you for the fundamental work

**Comment by everyoneishappy** [comment]
Yeah that's a nice idea.  Have you cheeked out Dist_Tesselator in the meantime?

**Comment by ggml** [comment]
yes, i found it useful for instances.
